ext/rotoscope/rotoscope.c

```diff
@@ -357,6 +365,8 @@ VALUE rotoscope_state(VALUE self) {
 
 void Init_rotoscope(void) {
   cTracePoint = rb_const_get(rb_cObject, rb_intern("TracePoint"));
+  cThread = rb_const_get(rb_cObject, rb_intern("Thread"));
```

Comment:
By using `rb_thread_current ` above you don't need this or `currentId`, but for future reference you can just use `rb_cThread`